### DISCUSSION TITLE
Installing DBeaver Office Support extension globally for all users

### DISCUSSION THREAD

**Discussion by Gnad1306** [dbeaver/dbeaver#22246] [Q&A]
Hello,

I'm using DBeaver community version 23.2.3. I installed the DBeaver Office Support extension so that I can export data as XLSX file. I installed the extension and when export data, option XLSX format appear and I'm able to export successful. However, it only effect for the user which is I used for installing the extension, for the others, they don't see the option XLSX format. And I did check the Installed Software on that user, they actually don't have DBeaver Office Support installed.

Please help me that is it possible for installing that extension globally which is effect for all user on same machine? Otherwise, is there any workaround for this kind of issue :(

Thank you in advance

**Comment by LonwoLonwo** [comment]
Hello @Gnad1306 

It is unclear from your message - is your DBeaver version installed for all versions or only for your user?

**Reply by Gnad1306** [reply]
Hi @LonwoLonwo, it was installed for all users. I mentioned I check for DBeaver Office Support extension in DBeaver on another user and it wasn't there
